Exercises/NoModules/Chapter 15/Soln15_01/Animals.cpp

```diff
@@ -0,0 +1,32 @@
+// Implementations of the Animal class and classes derived from Animal
+
+#include "Animals.h"
+
+// Constructor
+Animal::Animal(std::string_view name, unsigned weight) 
+   : m_name(name), m_weight(weight)
+{}
+
+// Return string describing the animal
+std::string Animal::who() const
+{
+  return "My name is " + m_name + ". My weight is " + std::to_string(m_weight) + " lbs.";
+}
+
+// Make like a sheep
```

Comment:
"make like a sheep"?
gramatically not perfect.
same for others.

Reply:
Then you need to check your grammar: "to make like" is a perfectly valid English idiom that means "to behave as". See https://idioms.thefreedictionary.com/make+like

Reply:
right, I just thought you forgot "sound", as in "Make sound like a sheep". ;)
but ok, seems to be fine then :)